modules/kmm-configuring-out-of-tree-module-images.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="kmm-configuring-out-of-tree-module-images_{context}"]
+= Configuring out-of-tree module images
+
+If you are using out-of-tree (OOT) modules, you can use the image configuration settings in OpenShift's core image configuration (`image.config.openshift.io/cluster`) to configure the modules instead of configuring each module configuration. 
+
+To configure the registry settings to allow for insecure registries for the OOT modules, use the configuration instructions in link:https://docs.openshift.com/container-platform/4.17/openshift_images/image-configuration.html#images-configuration-file_image-configuration[Configuring image registry settings].
+
+
+[NOTE]
+====
+The `registryTLS`, `baseImageRegistryTLS`, and `unsignedImageRegistryTLS` parameters are still used within the `Module` CR for validation purposes.
```

Comment:
@ybettan Note  in the section that follows (https://docs.openshift.com/container-platform/4.16/hardware_enablement/kmm-kernel-module-management.html#kmm-example-cr_kernel-module-management-operator) the `unsignedImageRegistryTLS` parameter is not used in the Module CR. Do we need to add it?

Reply:
I would remove it, if it is not used. and leave in the two that are.